_posts/2021-02-11-Creating-A-Game.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: Creating A Game
+description: 
+summary: 
+tags:
+---
+Today, I did typing club for about an hour. We also built a game, we had to create a new folder locally and work through a tutorial that Kajari provided for us we finished with a game. [This is the article we used](https://developer.mozilla.org/en-US/docs/Games/Tutorials/2D_Breakout_game_pure_JavaScript).
```

Comment:
Talk about pushing the code to a repository on github and what article you used to do that.

Lastly, talk about turning it into a github pages project and link directly to the game as well as the repository.